vendor/k8s.io/kubernetes/pkg/registry/policy/rest/storage_policy.go

```diff
@@ -55,11 +55,13 @@ func (p RESTStorageProvider) v1beta1Storage(apiResourceConfigSource serverstorag
 	storage["poddisruptionbudgets"] = poddisruptionbudgetStorage
 	storage["poddisruptionbudgets/status"] = poddisruptionbudgetStatusStorage
 
-	rest, err := pspstore.NewREST(restOptionsGetter)
```

Comment:
In the quest for the simplest patch, is this actually just wrapping it in an `if false` block?